readme.md

```diff
@@ -65,11 +84,14 @@ Here are the available options. The defaults are shown to the right of the optio
 - **password:** string (*required*)  
     The password for logging in to the developer portal on the target Roku device
 
+- **signingPassword:** string (*required for signing*)  
+    The password used for creating signed packages
+
 - **outDir?:** string = `"./out"`  
-    A full path to the folder where the zip package should be placed  
+    A full path to the folder where the zip/pkg package should be placed
 
-- **outFile?:** string = `"roku-deploy.zip"`  
-    The name the zip file should be given.  
+- **outFile?:** string = `"roku-deploy"`  
+    Basename the zip/pkg file should be given..
```

Comment:
Maybe say something like "The base file name that the zip/pkg should be given, without the extension."